trackintel/preprocessing/positionfixes.py

```diff
@@ -164,7 +159,11 @@ def generate_staypoints(
     # user_id of sp should be the same as ret_pfs
     sp["user_id"] = sp["user_id"].astype(pfs["user_id"].dtype)
 
-    return pfs, sp
+    if len(sp) <= 0:
```

Comment:
How could it be `<0`? should it rather `==` for readability? (Similarly for other changes)